.github/workflows/sometimes-workflow.yaml

```diff
@@ -0,0 +1,85 @@
+name: Sometimes
+run-name: ${{ github.event.pull_request.title }} - ${{ github.event.pull_request.head.sha }}
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: sometimes-node-modules
+      - uses: actions/setup-node@v3
+        with:
+          node-version: latest
+      - run: npm install
+  test:
+    needs: build
+    runs-on: ubuntu-latest
```

Comment:
put runs-on at the job level

Reply:
i think each job needs that but i will try